config.go

```diff
@@ -549,6 +551,63 @@ func (s *Settings) GetSender(ctx context.Context, env Environment) (send.Sender,
 	return send.NewConfiguredMultiSender(senders...), nil
 }
 
+// getGithubAppAuth returns app id and app private key if it exists.
+// returns false if it does not have both of those fields.
+func (s *Settings) getGithubAppAuth() (int64, []byte, bool) {
```

Comment:
It seems redundant to return a bool when you could just check for a non-zero result.